### ISSUE TITLE
More lag than default agar.

### ISSUE THREAD

**Issue by purrowler** [RealDebugMonkey/ZeachCobbler#13]
Default agar has a lot of problems with not being able to move but with this mod you keep standing still for more than ~1 min (i stayed alive that long without being able to move). maybe add the alt+r feature from agariomods?

**Comment by RealDebugMonkey** [comment]
I haven't been able to reproduce. What at are the circumstances -- using the grazer? Only at large sizes? Or perhaps only after playing a long time?

The issue sounds more like the issue Zeach is talking about here that can happen to pretty much anyone at any time. https://www.reddit.com/r/Agario/comments/3b4mm3/what_happened_to_this_game_do_you_need_fiber/csistp6?context=3

Do you see the same issue with Agariomods and does the alt+r feature fix the lag? If so I'll merge it for you.

**Comment by purrowler** [comment]
Yes its that issue and its not only if you are big or play longer time as seen here: http://gfycat.com/SelfishOddGermanpinscher 
I just connected and instantly wasn't able to move.
Yes i have the same problem with agariomods and yes alt+r fixes it but only for a period of time.

**Comment by RealDebugMonkey** [comment]
Will merge alt+r feature from agariomods in hope of addressing

**Comment by purrowler** [comment]
k thanks
